### PR TITLE
Update PELUX psplash logo

### PR DESCRIPTION
**Pull request by DunderRoffe**
Change psplash logo to a one that looks good in both portrait and
landscape mode.

### CODE REVIEW

File: recipes-core/psplash/psplash/psplash-start.service [hunk 1]
```diff
@@ -5,7 +5,7 @@ After=systemd-vconsole-setup.service systemd-udev-trigger.service systemd-udevd.
 DefaultDependencies=no
 
 [Service]
-ExecStart=/usr/bin/psplash --angle 270
+ExecStart=/usr/bin/psplash
```
Comment (martin-ejdestig) [inline recipes-core/psplash/psplash/psplash-start.service]: OK to remove because you rotated the image itself?

Comment (DunderRoffe) [reply recipes-core/psplash/psplash/psplash-start.service]: The image is symetrical. It looks identical from all sides.

Comment (DunderRoffe) [reply recipes-core/psplash/psplash/psplash-start.service]: Still testing though ... plz wait with merging

Comment (martin-ejdestig) [reply recipes-core/psplash/psplash/psplash-start.service]: Commit message updated with this info so resolving.

**Comment by DunderRoffe** [comment]
Tested and seems to work nice

**Comment by DunderRoffe** [comment]
> Tested and seems to work nice

Nvm could do some more tweaking

**Comment by JEderonn** [comment]
If you consider tweaking is finished, pl merge.

**Comment by DunderRoffe** [comment]
Tweaking done

**Pull request merged**